src/components/ContentReport/ContentReportForm.vue

```diff
@@ -139,4 +137,10 @@ export default {
   font-size: 13px;
   font-family: Source Sans Pro;
 }
+#back-button:hover{
+  color:rgb(73, 73, 73);
+}
+#cross-icon:hover{
+  background-color: grey;
```

Comment:
the issue says the text color needs to change on hover, not the background color